### DISCUSSION TITLE
deck.gl TextLayer is not displayed in Google Maps ( new GoogleMapsOverlay )

### DISCUSSION THREAD

**Discussion by billjoe** [visgl/deck.gl#7607] [Q&A] [answered]
Could someone please help me why the TextLayer is not displayed here in my example? Thank you!
https://codesandbox.io/s/bitter-snowflake-0oxnqw

**Comment by Pessimistress** [comment, marked as answer]
TextLayer's data prop must be iterable. It does not accept an arbitrary GeoJSON. See GeoJsonLayer's documentation if you want to render text labels from a GeoJSON.
